games-strategy/spring/spring-104.0.ebuild

```diff
@@ -8,7 +8,7 @@ if [[ $PV = 9999* || $PV = *_rc* ]]; then
 	GIT_ECLASS="git-r3"
 	EGIT_REPO_URI="https://github.com/spring/spring.git"
 	EGIT_BRANCH="develop"
-	EGIT_COMMIT="37dc5346ca121d38ecb451e556ac1ac655f14ca4" # 1222, 104 rc 1
+	EGIT_COMMIT="103.0"
```

Comment:
that should be 104.0 i guess :)

Reply:
It does not matter as that section only applies to repo checkouts, i.e. _rc* and -9999, not releases

Reply:
but I should change it to something that is guaranteed to error out

Reply:
that is, if someone copied the ebuild to whatever and expected it to work out of the box.

Reply:
IMO it should be only set when $PV = *_rc*

Reply:
Sure, for -9999 it is simply deleted, but the rest of the settings still apply.

Reply:
i don't care, but 103.0 is the worst value you could fill in there :-)

just revert the change is IMO best then.

Reply:
I cannot revert it, that diff was generated by adding 104 ebuild copied from 103 and removing 104_rc1. Then git made its life easier by seemingly renaming 104_rc1 to 104 and changing a few bits.